test/framework/mesh/logical_volume/lv_lua_func.lua

```diff
@@ -0,0 +1,37 @@
+--[[# test for lus function used as a logical volume
+--]]
```

Comment:
```suggestion
-- test for lua function used as a logical volume
```
If this is a block comment, why use it for one line of text?